src/service/models.py

```diff
@@ -0,0 +1,45 @@
+"""
+Pydantic models for the Spark Manager API.
+"""
+
+import re
+
+from pydantic import BaseModel, Field, field_validator
+
+
+class SparkClusterConfig(BaseModel):
+    """Configuration for creating a new Spark cluster."""
+
+    worker_count: int = Field(default=2, description="Number of worker nodes")
```

Comment:
The recommended way to do this now is with `Annotated`

Reply:
In this case it's fine as is, but Annotated is the recommended way at this point, so I've been using it everywhere for consistency

Reply:
👍